### PR TITLE
Attila dashboard

### PR DESCRIPTION
**Pull request by h-attila**
some dashboard design added...

### CODE REVIEW

File: web/dashboard.html [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
-          <li ng-repeat="contact in allContacts"><p class="dashboard-p">ismerős: {{ contact.name }}, ki ez: {{ contact.description }}, kinek az ism.: {{ contact.user.name }}, id: {{ contact.user.id }}, </p></li>
-        </ul>
+  <div class="konnekt-logo-background">
+    <div class="konnekt-logo-dashboard"></div>
+  </div>
+
+  <div class="dashboard-container">
+    <div class="dashboard-table">
+      <div class="add-new-contact"></div>
+        <div class="table-header">
```
Comment (hubudibu) [inline web/dashboard.html]: if it's a table, try using a `table` element!

File: web/index.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8">
     <title>Konnekt Kontakt Manager</title>
     <link href="https://fonts.googleapis.com/css?family=Merriweather:400i" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Raleway:800" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway:400,800" rel="stylesheet">
```
Comment (hubudibu) [inline web/index.html]: you can import multiple fonts in one link tag from google fonts and thus reduce the number of requests: `https://fonts.googleapis.com/css?family=Merriweather:400i|Raleway:400,800`

File: web/dashboard.html [hunk 3]
```diff
@@ -1,10 +1,47 @@
...
-    <div class="dashboard">
-      <div class="dashboard-list">
-        <ul>
-          <li ng-repeat="contact in allContacts"><p class="dashboard-p">ismerős: {{ contact.name }}, ki ez: {{ contact.description }}, kinek az ism.: {{ contact.user.name }}, id: {{ contact.user.id }}, </p></li>
-        </ul>
+  <div class="konnekt-logo-background">
+    <div class="konnekt-logo-dashboard"></div>
+  </div>
+
+  <div class="dashboard-container">
```
Comment (hubudibu) [inline web/dashboard.html]: use a `table` element for tabled data

File: web/src/app.js [hunk 4]
```diff
@@ -198,6 +196,33 @@ konnektApp.factory('UserService', ['HttpService', '$window', function (HttpServi
 }]);
 
 
+konnektApp.factory('DataHandling', ['HttpService', function (HttpService) {
```
Comment (hubudibu) [inline web/src/app.js]: what kind of data? as I see it's all about contacts, let's call it something like that. data is way too broad

**Review by hubudibu — CHANGES_REQUESTED** [review]
+ remove both `newcontact` images, use a button tag with real text instead, + an svg icon for the + sign

**Comment by h-attila** [comment]
conflicts resolved

File: web/index.html (hunk 2 above)
Comment (h-attila) [reply web/index.html]: cool! :)

**Pull request merged**